src/routes/home/Home.js

```diff
@@ -648,9 +652,12 @@ class Home extends React.Component {
       this.attachmentPlates.set(attachmentFile, result);
       return result;
     } catch (err) {
+      if (err == 'license plate (image was read, but no plates were found)') {
```

Comment:
Instead of copy/pasting this exact error message, we should probably make a constant for it.